src/design-system/stories/InputField.stories.tsx

```diff
@@ -90,10 +93,18 @@ type Story = StoryObj<typeof meta>
 
 const InputFieldTemplate = (args: InputFieldProps) => {
   const [value, setValue] = useState(args.value)
+
+  const handleClearInput = () => {
+    setValue('')
+  }
```

Comment:
Consider adding a space between event handlers.
```suggestion
  }
  
```